src/Teams/Controllers/MultipleAnswerQuestionController.cs

```diff
@@ -37,5 +39,47 @@ public IActionResult MultipleAnswerQuestionForm(string id, int[] answers)
             };
             return View(question);
         }
+
+        public IActionResult EditMultipleAnswerQuestion(Guid id)
+        {
+            var question = new MultipleAnswerQuestionViewModel()
+            {
+                SourceQuestion = questionRepository.PickById(id),
+            };
+            return View(question);
+        }
+
+        public IActionResult AddMultipleAnswerQuestion()
+        {
+            return View();
+        }
+
+        [HttpPost]
+        public IActionResult AddMultipleAnswerQuestion([FromBody] MultipleQuestionAddModel fromAjax)
```

Comment:
Renamed fromAjax to multipleAnswersQuestionDTO. I think it's more meaningful. Then I use DTO instead model 
because as far as I know it's the same in this case.